### DISCUSSION TITLE
Can I use a default variable value through the 'ais-refinement-list'?

### DISCUSSION THREAD

**Discussion by albericobenhur** [algolia/instantsearch#5476] [Q&A] [answered]
I'm trying to use the url path to bring a specific result depending on the route I accessed, but I'm having trouble identifying which tag or which attribute I should use to get that result

**Comment by Haroenv** [comment, marked as answer]
Have you tried routing? https://www.algolia.com/doc/guides/building-search-ui/going-further/routing-urls/vue/

**Reply by albericobenhur** [reply]
it worked for me thanks :smiley:
